constants/constants.go

```diff
@@ -2,9 +2,9 @@ package constants
 
 // provider types
 const (
-	PROVIDER_TYPE_SFTP     int = 0
-	PROVIDER_TYPE_GDRIVE   int = 1
-	PROVIDER_TYPE_ONEDRIVE int = 2
+	PROVIDER_TYPE_SFTP     int = 1
+	PROVIDER_TYPE_GDRIVE   int = 2
+	PROVIDER_TYPE_ONEDRIVE int = 3
```

Comment:
dlaczego tutaj odeszliśmy od 0?

Reply:
Gorm gdy nie znajdzie obiektu w bazie danych podczas seedowania to ustawia pusty obiekt jako Provider (który ma m.in. typ 0). Jeżeli zaczniemy numerację od 1, to łatwo będziemy mogli rozpoznać, że obiekt jest pusty, bo nie został znaleziony w BD.

Reply:
ack